pkgs/applications/misc/gsctl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "gsctl-${version}";
+  version = "0.15.1";
+
+  goPackagePath = "github.com/giantswarm/gsctl";
+
+  src = fetchFromGitHub {
+    owner = "giantswarm";
+    repo = "gsctl";
```

Comment:
Can you use ``pname`` instead of name and use it here?

Reply:
Done. Look better?

Reply:
Yes, thank you!